packages/tldraw/src/lib/ui/context/breakpoints.tsx

```diff
@@ -12,13 +12,13 @@ export interface BreakPointProviderProps {
 
 /** @public @react */
 export function BreakPointProvider({ forceMobile = false, children }: BreakPointProviderProps) {
-	const editor = useEditor()
+	const editor = useMaybeEditor()
 
 	const breakpoint = useValue(
 		'breakpoint',
 		() => {
 			// This will recompute the viewport screen bounds changes...
-			const { width } = editor.getViewportScreenBounds()
+			const { width } = editor?.getViewportScreenBounds() ?? { width: window.innerWidth }
```

Comment:
i think window is probably a safer bet. remember this fallback is only used if the editor is not available, and if the editor is not available the container might not be available, and even if the container is available it might not be correctly sized.